src/gei/Commands/MigrateRepo/MigrateRepoCommand.cs

```diff
@@ -188,6 +189,8 @@ public override MigrateRepoCommandHandler BuildHandler(MigrateRepoCommandArgs ar
             var ghesVersionChecker = ghesVersionCheckerFactory.Create(ghesApi);
             var warningsCountLogger = sp.GetRequiredService<WarningsCountLogger>();
 
+            log.AddRedactionPattern(new Regex("\\?token=[A-Z0-9]{29}"));
```

Comment:
Two things:
1. I would move this into the`OctoLogger` itself.
2. this will also replace the `token=`, what we preferably want is to just mask the secret, for that the regex pattern needs to be a bit more complex, like this --> `\b(?<=token=)(.+?)\b`, I used look behind (`?<=`) in order to not replace the `token=`.
3. Even if you want to keep this logic (instead of moving the pattern into `OctoLogger`), I would do the registration inside the handler and not in here. The `BuildHandler`'s purpose is to just create the handler and nothing else.